src/main/java/uk/co/companieshouse/kafka/error/config/KafkaConsumerConfig.java

```diff
@@ -0,0 +1,73 @@
+package uk.co.companieshouse.kafka.error.config;
+
+import java.util.HashMap;
+import java.util.Map;
+import org.apache.kafka.clients.consumer.ConsumerConfig;
+import org.apache.kafka.common.serialization.ByteArrayDeserializer;
+import org.apache.kafka.common.serialization.StringDeserializer;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.kafka.annotation.EnableKafka;
+import org.springframework.kafka.config.ConcurrentKafkaListenerContainerFactory;
+import org.springframework.kafka.core.ConsumerFactory;
+import org.springframework.kafka.core.DefaultKafkaConsumerFactory;
+import org.springframework.kafka.core.KafkaTemplate;
+import uk.gov.companieshouse.logging.Logger;
+
+/**
+ * Configuration required for creating the consumers
+ * Requires configuration beans from KafkaSerializerConfig
+ */
+@EnableKafka
+@Configuration
+public class KafkaConsumerConfig {
+
+    @Autowired
+    KafkaErrorProperties kafkaErrorProperties;
+
+    @Autowired
+    private Logger logger;
+
+    @Autowired
+    private KafkaTemplate kafkaTemplate;
+
+    @Bean
+    public ConsumerFactory<String, byte[]> consumerFactory() {
+        Map<String, Object> props = new HashMap<>();
+        props.put(
+                ConsumerConfig.BOOTSTRAP_SERVERS_CONFIG,
+                kafkaErrorProperties.getBootstrapAddress());
```

Comment:
The name of the config variable is plural - can Kafka find all brokers in the cluster from one bootstrap address?

Reply:
Afaik, the bootstrap address is a constant entry point and the client finds all of the brokers in a cluster via that entry point. We don't need to specify all of the brokers here. I think it is a list so you can specify >1 broker, in case the first broker is down.